src/gerbil/gxi

```diff
@@ -1,11 +1,48 @@
 #!/bin/sh
 set -eu
 
+
 if [ -z "${GERBIL_HOME:-}" ]; then
-    GERBIL_HOME=$(dirname $(cd ${0%/*} && echo $PWD))
```

Comment:
quotes missing around the internal `$()`

Reply:
AND around the `${}`. Around the `$PWD` just to be sure, too.

Reply:
Definitely around the $PWD ... it could contain consecutive spaces.

Reply:
awesome, posted the link to this file before scrolling down lol.

I'll fix that issue.